app/models/asset.rb

```diff
@@ -197,7 +197,7 @@ def barcode_formatted_for_printing
     barcode
   end
 
-  def printable_object(username = 'unknown')
```

Comment:
I could have taken this change further, as it also removes the need to include username in `#print` but then that cascades still further. Given we still need to load the user for their print settings, there weren't going to be any major gains taking this further.